thread/src/components/common/index.js

```diff
@@ -0,0 +1,3 @@
+export * from './CommonButton';
```

Comment:
In each of your components export as you would normally.
i.e. `export default CommonButton;`

Then in this file export like this.
`export { default as CommonButton } from './CommonButton';`